docs/doc/10-deploy/06-metasrv/70-metasrv-cli-api.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Databend Meta CLI
+sidebar_label: Meta Service Command Line API
+description:
+    Access Databend Meta Service Cluster with Command Line Interface
+---
+
+The binary `databend-meta` provides several handy commands to access the KVApi of databend meta service.
+`databend-meta --help` also includes a simple guide on using these CLI commands.
+
+The command line API is limited that:
+- only string key and string value are supported.
+- `seq` is not supported.
+
```

Comment:
Great!
It would be great to add some hints, and tell the user we do not recommend changing the metasrv data in this way, and what cases we need to use this API :)

Reply:
Resolve it and track it in another issue